code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -360,3 +360,29 @@
 	var/damage = ((min_damage / 4) + (max_damage / 4)) / 2 // We expect you to have 4 functional limbs- if you have fewer you're probably not going to be so good at lifting
 
 	return ceil(damage * (ceil(athletics_level / 2)) * fitness_modifier * maxHealth)
+
+/mob/living/carbon/human/proc/item_heal(mob/living/carbon/human/human, mob/user, brute_heal, burn_heal, heal_message_brute, heal_message_burn, required_bodytype,)
```

Comment:
Is the first argument here semantically identical to `src`? Why do we need to pass that in if the proc is called on this human?